src/Microsoft.VisualStudio.Editor.Razor/InternalVisualStudioDocumentTracker.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal abstract class InternalVisualStudioDocumentTracker : VisualStudioDocumentTracker
```

Comment:
Do we need this? Can we just use the **implementation**. These methods aren't an abstraction, they are part of an implementation.